cmd/syncthing/main.go

```diff
@@ -815,6 +815,7 @@ func syncthingMain(runtimeOptions RuntimeOptions) {
 	// Add and start folders
 	for _, folderCfg := range cfg.Folders() {
 		if folderCfg.Paused {
+			folderCfg.CreateRoot()
```

Comment:
@imsodin Happened to be looking back at this... This is a problem if we're starting up, a folder is paused, and the folder is unmounted or otherwise missing. We will then create a new, empty, root for it. If the folder is then unpaused, we'll assume it became empty and sync a complete delete of it...

Reply:
@calmh I seem to remember that this came up before but was considered not a problem, as the missing folder marker prevents it?

Reply:
CreateRoot does not create the marker though?

Reply:
It doesn't, that's done by `cfg.CreateMarker`.